app/graphql/types/icon_id_to_name_type.rb

```diff
@@ -0,0 +1,20 @@
+# Reference: https://openweathermap.org/weather-conditions#Weather-Condition-Codes-2
+
+class Types::IconIdToNameType < GraphQL::Schema::Scalar
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Not something to be concerned about for this PR, but this reminds me to evaluate the usefulness of the `CyclomaticComplexity` rule...